mozilla-release/toolkit/components/url-classifier/SafeBrowsing.jsm

```diff
@@ -153,12 +153,15 @@ this.SafeBrowsing = {
     switch (kind) {
       case "Phish":
         pref = "browser.safebrowsing.reportPhishURL";
+        Services.telemetry.getHistogramById("REPORT_DECEPTIVE_SITE").add(0);
```

Comment:
@luciancor (0, 1, 2) mean ('Phish', 'PhishMistake',  'MalwareMistake'). We have three kind of signals. Since we can only accumulate the signals by numbers, we need to map these strings in to integer values.